tpl/pbs_mess_uq.tpl

```diff
@@ -1,2 +0,0 @@
-cd ${{PBS_O_WORKDIR}}/me
```

Comment:
file deleted because all mess files will have a numbering regardless of whether or not UQ is turned on to simpliify things.